[["file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/symfony/http-kernel/Exception/UnauthorizedHttpException.php",{"_uri":"file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/symfony/http-kernel/Exception/UnauthorizedHttpException.php","_root":{"kind":4096,"name":"file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/symfony/http-kernel/Exception/UnauthorizedHttpException.php","location":{"uriHash":1443598072,"range":{"start":{"line":0,"character":0},"end":{"line":31,"character":0}}},"children":[{"kind":512,"name":"Symfony\\Component\\HttpKernel\\Exception","location":{"uriHash":1443598072,"range":{"start":{"line":11,"character":0},"end":{"line":11,"character":49}}}},{"kind":1,"name":"Symfony\\Component\\HttpKernel\\Exception\\UnauthorizedHttpException","location":{"uriHash":1443598072,"range":{"start":{"line":16,"character":0},"end":{"line":30,"character":1}}},"children":[{"kind":32,"name":"__construct","location":{"uriHash":1443598072,"range":{"start":{"line":24,"character":4},"end":{"line":29,"character":5}}},"doc":{"description":"","type":""},"modifiers":1,"type":"","children":[{"kind":128,"name":"$challenge","location":{"uriHash":1443598072,"range":{"start":{"line":24,"character":32},"end":{"line":24,"character":42}}},"doc":{"description":"WWW-Authenticate challenge string","type":"string"},"scope":"__construct"},{"kind":128,"name":"$message","location":{"uriHash":1443598072,"range":{"start":{"line":24,"character":44},"end":{"line":24,"character":59}}},"doc":{"description":"The internal exception message","type":"string"},"value":"null","scope":"__construct"},{"kind":128,"name":"$previous","location":{"uriHash":1443598072,"range":{"start":{"line":24,"character":61},"end":{"line":24,"character":88}}},"type":"Exception","doc":{"description":"The previous exception","type":"Exception"},"value":"null","scope":"__construct"},{"kind":128,"name":"$code","location":{"uriHash":1443598072,"range":{"start":{"line":24,"character":90},"end":{"line":24,"character":99}}},"doc":{"description":"The internal exception code","type":"int"},"value":"0","scope":"__construct"}],"scope":"Symfony\\Component\\HttpKernel\\Exception\\UnauthorizedHttpException"}],"associated":[{"kind":1,"name":"Symfony\\Component\\HttpKernel\\Exception\\HttpException"}],"modifiers":0}]},"_hash":1443598072}]]